pom.xml

```diff
@@ -30,6 +30,9 @@
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
       <version>4.18.1</version>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>4.18.1</version>
```

Comment:
**Issue**: Duplicate dependency entries for 'selenium-java' which can lead to confusion and potential build issues. <br> **Fix**: Remove the duplicate dependency entry to ensure clarity and prevent potential conflicts during the build process. <br> **Code Suggestion**: 
 ```
 -      <groupId>org.seleniumhq.selenium</groupId>
 -      <artifactId>selenium-java</artifactId>
 -      <version>4.18.1</version>
 ```